ReactAndroid/src/main/java/com/facebook/react/uimanager/layoutanimation/LayoutAnimationController.java

```diff
@@ -91,4 +100,39 @@ public void applyLayoutUpdate(View view, int x, int y, int width, int height) {
       view.startAnimation(animation);
     }
   }
+
+  /**
+   * Animate a view deletion using the layout animation configuration supplied during initialization.
+   *
+   * @param view     The view to animate.
+   * @param callback Called once the animation is finished, should be used to
+   *                 completely remove the view.
+   */
+  public void deleteView(final View view, final Callback callback) {
```

Comment:
The reason I had to make a separate method for delete is that I need to hook into the `manageChildren` method of `NativeViewHierarchyManager` while `applyLayoutUpdate` is called in `updateLayout` of `NativeViewHierarchyManager` which is not called for view deletions that don't change it's parent size.